test/vm.install

```diff
@@ -2,6 +2,10 @@
 
 SRPM="$1"
 
+# always remove older versions of these RPMs if they exist
+# to ensure newly built packages have been installed
+yum -y remove lorax lorax-composer composer-cli
```

Comment:
Does this make the `rpm -e ...` further down obsolete? Seems like it's doing the same thing in a less general way.

Reply:
Indeed you are right. I've removed the rpm -e and left the explicit yum remove